PowerEditor/src/MISC/PluginsManager/PluginsManager.cpp

```diff
@@ -664,8 +664,9 @@ void PluginsManager::runPluginCommand(size_t i)
 			}
 			catch (...)
 			{
-				TCHAR funcInfo[128];
-				generic_sprintf(funcInfo, TEXT("runPluginCommand(size_t i : %zd)"), i);

```

Comment:
Hmm... The signature of swprintf is indeed:
```cpp
int swprintf(
   wchar_t *buffer,
   size_t count,
   const wchar_t *format [,
   argument]...
);
```
according the [MS document](https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/sprintf-sprintf-l-swprintf-swprintf-l-swprintf-l?view=msvc-170)

I don't understand how the above code can be compiled.
Do you have any explanation?

Reply:
OK, I see. There are another swprintf function with no `size_t count,` as argument.